test/e2e-tests.js

```diff
@@ -505,6 +505,25 @@ lab.experiment("screenshot plugin", async () => {
       );
     }
   );
+
+  it(
+    "returns the screenshot info with correct cache-control headers without wait",
+    { timeout: 5000, plan: 5 },
+    async () => {
+      const response = await server.inject(
+        "/screenshot/mock-item-active.json?target=pub1&width=500"
```

Comment:
Why is it needed to pass the width as query parameter?- Is this a feature to let the server calculate the height based on a fixed width?

Reply:
yes, without the width we cannot know the height. As this screenshot feature is used primarily to make a PNG from a web based tool view, the width needs to be given. As it is with (every) web based layout: layout defines the width, content defines the height.